npm/npm.go

```diff
@@ -22,9 +23,10 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
-var (
-	hostNetAgentURLForNpm = "http://168.63.129.16/machine/plugins?comp=netagent&type=npmreport"
-	contentType           = "application/json"
+const (
+	hostNetAgentURLForNpm  = "http://168.63.129.16/machine/plugins?comp=netagent&type=npmreport"
+	contentType            = "application/json"
+	retryWaitTimeInSeconds = 60
```

Comment:
Is this a global retryWaitTime for npm? or just for telemetry? a more precise name will avoid wrong reuse

Reply:
Thanks for the comment. I have renamed to retryWaitTimeInSeconds to telemetryRetryWaitTimeInSeconds